com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -526,17 +549,41 @@ internal static void RpcReceiveQueueItem(ulong clientId, Stream stream, float re
                 return;
             }
 
-            ProfilerStatManager.rpcsRcvd.Record();
+            ProfilerStatManager.RpcsRcvd.Record();
             PerformanceDataManager.Increment(ProfilerConstants.NumberOfRPCsReceived);
 
-            var rpcQueueContainer = NetworkManager.Singleton.rpcQueueContainer;
+#if DEVELOPMENT_BUILD || UNITY_EDITOR
```

Comment:
This seems a bit odd - is this supposed to be here?

Reply:
this is coming from `NetworkManager.HandleIncomingData` and `NetworkManager.ReceiveCallback` methods, wrapping around `InternalMessageHandler.RpcReceiveQueuItem` method. those stats should've been inside this method, so I moved them here instead of leaving them over there. I can move them back but looked like fairly safe to move here — I can revert it back if you'd prefer.

Reply:
Ok